pyqt-apps/scripts/sirius-hla-bl-ap-manaca.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python-sirius
```

Comment:
@anacso17 and @fernandohds564 , I find it confusing the IOC having a device name like `SI-09SABL:AP-...`and the script for running the associated window being named `sirius-hla-bl-ap-manaca.py`... what do you think?

Reply:
How do you think we should name it, @xresende?

Reply:
Maybe surius-hla-si-ap-manaca?

Reply:
If we change the name of the IOC to `bl-ap-manaca`, I think the best device name would be `BL-09SA:AP-Manaca-MVS2` and, accordingly, we should use `BL-09SA: MP-Manaca` forthe Interlock Status IOC... These names are more similar to the names that prof. Rubens adopted for the diagnostic lines... What do you think?

Reply:
It is easier to use `si` instead of `bl` (requires less change of already committed code at this point). In any case, ana, since we last discussed beamline names I had in mind contacting prof. rubens about changing his naming to our new standard, that is, using `si`... 

I would use `si` but don't see any problem with `bl`, as long as we do it consistently.